module-4/backend/week_three.md

```diff
@@ -5,15 +5,33 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+A tool used to run server-side JS. It is single process. 
 2. What is Express? What is Express similar to in the Ruby world?
+Express is a framework for Node.js. It is similar to ruby in its web application building capabilities. 
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+app.route('/muffins')
+    .get(foodList.list_all_muffins)
+    .post(foodList.create_a_muffin);
 4. What do we use Knex for?
+Knex is used for SQL queries within JS. 
 5. How could you organize your code to follow the MVC design pattern in your Quantified Self project?
+Move all queries to teh database to a model file. Move the routes to a controller.
 6. How do you execute raw SQL in node?
+database.raw
+(SQL)
 7. What are some advantages to having your front end and back end code live in separate applications? What are some disadvantages?
+Advantages would be that it allows for easier expanding. The scalibility is much easier. 
+Disadvantages would be that they have to be able to communicate clearly between eachother. 
 
 #### Review  
 
 8. Describe DNS.
+A hierarchical naming system. Sort of like a phone book.
 9. What does writing clean code mean to you?
+Code that could be understood by someone who wasn't involved with the project. 
 10. If you were building an application that models hotels, what classes would you need? What classes would be responsible for what functionality?
+
```

Comment:
this is a good start, but you didn't touch on the second question - "what classes would be responsible for what functionality?"